scripts/generate-sdk/generate-sdk.sh

```diff
@@ -209,10 +205,12 @@ for service_json in ${ROOT_DIR}/oas/*.json; do
 done
 
 # Add examples to workspace
-for example_dir in ${SDK_REPO_LOCAL_PATH}/examples/*; do
-    cd ${example_dir}
-    go work use .
-done
+if [ -d ${SDK_REPO_LOCAL_PATH}/examples ]; then
+    for example_dir in ${SDK_REPO_LOCAL_PATH}/examples/*; do
+        cd ${example_dir}
+        go work use .
+    done
+fi
 
 # Add scripts to workspace
 cd ${SCRIPTS_FOLDER}
```

Comment:
Might as well add a check here right?

Reply:
Also, why is this a variable and for instance the examples folder above is hard coded?

Reply:
Good point, added it. Same for `SERVICES_FOLDER`